deducing-this.md

```diff
@@ -370,24 +458,25 @@ If references and pointers do not have the same representation for member functi
 
 This matters because code written in the "`this` is a pointer" syntax with the `this->` notation needs to be assembly-identical to code written with the `self.` notation; the two are just different ways to implement a function with the same signature.
 
-## Interplays with capturing `[this]` and `[*this]`
+## `virtual` and `this` as value
 
-`this` just designates the parameter that is bound to the reference to the function object. It does not, in any way, change the meaning of `this`.
+Virtual member functions are always dispatched based on the type of the object the dot -- or arrow, in case of pointer -- operator is being used on. Once the member function is located, the parameter `this` is constructed with the appropriate move or copy constructor and passed as the `this` parameter, which might incur slicing.
 
-If other language features play with what `this` means, they are completely orthogonal and do not have interplays with this proposal. However, it should be obvious that develpers have a great potential for introducing hard-to-read code if they are at all changing the meaning of `this`, especially in conjunction with this proposal.
+Effectively, there is no change from current behavior -- only a slight addition of a new overload that behaves the way a user would expect.
 
-## Is `auto&& this self` allowed in member functions as well as lambdas?
+Virtual member function templates are not allowed in the language today, and this paper does not propose a change from the current behavior for member functions with explicit object parameters either.
 
-Yes. `auto&& param_name` has a well-defined meaning that is unified across the language. There is absolutely no reason to make it less so.
+## Can `static` member functions have a `this` parameter?
+
+No. Static member functions currently do not have an implicit `this` parameter, and therefore have no reason to have an explicit one.
 
-## Impact on the Standard
+## Interplays with capturing `[this]` and `[*this]`
 
-TBD: A bunch of stuff in section 8.1.5 [expr.prim.lambda].
+`this` just designates the parameter that is bound to the reference to the function object. It does not, in any way, change the meaning of `this`.
```

Comment:
In the parameter list, `this` just designates... ... change the meaning of `this` in the body of a function.